tests/integration/acp/dac/add_policy/with_empty_args_test.go

```diff
@@ -41,28 +41,15 @@ func TestACP_AddPolicy_EmptyPolicyCreator_Error(t *testing.T) {
 				Identity: testUtils.NoIdentity(),
 
 				Policy: `
-                    name: test
-                    description: a basic policy that satisfies minimum DRI requirements
-
-                    actor:
-                      name: actor
-
-                    resources:
-                      users:
-                        permissions:
-                          read:
-                            expr: owner
-                          update:
-                            expr: owner
-                          delete:
-                            expr: owner
-
-                        relations:
-                          owner:
-                            types:
-                              - actor
-
-                `,
+description: a basic policy that satisfies minimum DRI requirements
+name: test
+resources:
+- name: users
+  permissions:
+  - name: delete
+  - name: read
+  - name: update
```

Comment:
question: So this was not possible because the `expr` couldn't be empty under the nestings?
```suggestion
  permissions:
  - delete:
  - read:
  - update:
```

Reply:
It was possible but it "felt" wrong. 
More than once, we noticed that new users couldn't tell apart which keywords in a policy were reserved and what was user defined. The `name` field makes that explicit. As consequence, explicitly setting the permission `name` made omitting the `expr` field less awkward.